src/zenml/integrations/evidently/steps/evidently_profile.py

```diff
@@ -150,6 +159,39 @@ def entrypoint(  # type: ignore[override]
             EvidentlyDataValidator.get_active_data_validator(),
         )
         column_mapping = None
+        if config.ignored_cols is not None and not (
```

Comment:
```suggestion
        if config.ignored_cols and not (
```

I think maybe you can refactor it down to this..